src/sagemaker/jumpstart/types.py

```diff
@@ -1917,6 +1917,12 @@ def from_manifest(self, studio_manifest_entry: Dict[str, Any]):
             self.datatype = studio_manifest_entry["dataType"]
         if studio_manifest_entry.get("license"):
             self.license = studio_manifest_entry["license"]
+        task_pattern = r"\| Task: \| (.+?)\|"
```

Comment:
this should be a constant, not a local variable

Reply:
thanks, updated